src/index.js

```diff
@@ -157,6 +161,13 @@ function Swarm (peerInfo) {
           this.muxedConns[pi.id.toB58String()] = {}
           this.muxedConns[pi.id.toB58String()].muxer = muxedConn
           this.muxedConns[pi.id.toB58String()].conn = conn // to be able to extract addrs
+
+          self.emit('peer-mux-established', pi)
+
+          muxedConn.on('close', () => {
+            delete self.muxedConns[pi.id.toB58String()]
+            self.emit('peer-mux-closed', pi)
```

Comment:
Could we just call it `peer-disconnect`, as that feels like it would be easier to understand (and to remember)